local-test.env

```diff
@@ -1,6 +1,7 @@
 # Setups a local environment to simulate a Github Action run
-export GITHUB_TOKEN=1234
-export GITHUB_REPOSITORY=rymndhng/release-on-push-action
-export GITHUB_SHA=1234
+export GITHUB_TOKEN=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
+export GITHUB_REPOSITORY=denisgarciaglobal/release-on-push-action
+export GITHUB_SHA=ca1e899fad11f1c758a53ff9aa51294fc724fe6d
```

Comment:
This probably isn't intentional, but this probably shouldn't be commited in this PR